warehouse/query-core/src/main/java/datawave/query/composite/CompositeTerm.java

```diff
@@ -85,7 +89,17 @@ public void getNodesAndExpressions(List<Class<? extends JexlNode>> nodeClasses,
      * @return
      */
     private String getAppendedExpressions() {
-        return String.join(separator, expressionList);
+        final List<String> newExpressions;
+        if (Iterables.getLast(jexlNodeList) instanceof ASTERNode && !jexlNodeList.stream().allMatch(x -> x instanceof ASTERNode)) {
```

Comment:
Based on what you've said in the comments and the test added below, I think maybe this condition is wrong? This tests that the last node is an ASTERNode but not all of the nodes are ASTERNode. However, if the list had 3 items where 1st and 3rd are ASTERNode, then this condition would be true. Maybe you want the following?
```suggestion
        if (Iterables.getLast(jexlNodeList) instanceof ASTERNode && jexlNodeList.stream().limit(jexlNodeList.size() - 1).noneMatch(x -> x instanceof ASTERNode)) {
```

or

```suggestion
        if (Iterables.getLast(jexlNodeList) instanceof ASTERNode && jexlNodeList.stream().filter(x -> x instanceof ASTERNode).count() == 1) {
```